torch/ao/sparsity/parametrization.py

```diff
@@ -0,0 +1,10 @@
+from torch import nn
```

Comment:
We are implementing FakeSparsity by parameterization, I see this as the base-class that can be overridden by other implementations of FakeSparse (for example with learnable masks).
Suggest calling this file FakeSparse.py (similar to FakeQuantize.py)